### PR TITLE
Forum blog post 2022/01/11

### PR DESCRIPTION
**Pull request by maelle**
TODOS

- [x] Add (stat) software review category once we know where to encourage activity.

### CODE REVIEW

File: content/blog/2022-01-11-gather-on-ropensci-forum/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+slug: "ropensci-forum"
+title: Gather on rOpenSci Forum
+author:
+  - The rOpenSci Team
+date: 2022-01-11
+tags:
+  - forum
+  - help
+description: "Why and how participate in rOpenSci forum; description of current categories."
```
Comment (steffilazerte) [inline content/blog/2022-01-11-gather-on-ropensci-forum/index.md:10]:
```suggestion
description: "Why and how participate in the rOpenSci forum; descriptions of current categories."
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle!

Looks good. My main comment is that I *think* it's "**the** rOpenSci forms" (but I'm not 100% sure about that), and that I think the forum categories should be capitalized in the text (otherwise make the subheadings sentence case?).

Other than that, looks good!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

File: content/blog/2022-01-11-gather-on-ropensci-forum/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+* You can add the RSS feed of each category to your feed reader. The link to the RSS feed is the URL to the category plus `.rss` e.g. `https://discuss.ropensci.org/c/package-development/29.rss` (but if you copy-paste the URL `https://discuss.ropensci.org/c/package-development/29` in e.g. Feedly the actual feed will be found without your help).
+
+* In our [semi-open slack workspace](https://contributing.ropensci.org/resources.html#channels), there's a channel, `#discuss-ropensci` with all activity from the forum, but that's probably not optimal for casual forum use.
+
+* Subscribing to our [newsletter](/news) would mean you indirectly gain information from the forum as we summarized recent use cases, and digest package development tips.
+
+{{< figure src="cloud.jpg" alt="Thought cloud bubble on a pink background" caption="Picture by [Miguel Á. Padriñán on Pexels](https://www.pexels.com/fr-fr/@padrinan)." link="https://www.pexels.com/photo/white-bubble-illustration-1111372/" width="500px" class="center" >}}
+
+## Not forum categories
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2022-01-11-gather-on-ropensci-forum/index.md:93]: Good point!

**Comment by maelle** [comment]
@steffilazerte Thanks a lot for the review! :pray: I'll merge and tweet tomorrow.

**Pull request merged**